kronolith: LDAP-DNs in value-fields of group ACL dropdown (Bug #10692, Bug #15440)
     https://issues.kolab.org/issue4170

	 Without this patch LDAP-DNs appear in the value-fields oder the group-ACL dropdown
	 this seems to have cause problems with setting group ACLs on the folders

--- a/horde-webmailer/kronolith/templates/perms/perms.inc.orig	2008-03-17 17:26:39.000000000 +0100
+++ b/horde-webmailer/kronolith/templates/perms/perms.inc	2008-03-17 17:28:12.000000000 +0100
@@ -262,9 +262,9 @@
   <td class="header" align="center"><?php echo _("Delegate") ?></td>
   <td class="header">&nbsp;</td>
 </tr>
-<?php $groupperms = $perm->getGroupPermissions(); foreach ($groupperms as $group => $gperm): if (!$groups->exists($groups->getGroupName($group))) continue; ?>
+<?php $groupperms = $perm->getGroupPermissions(); foreach ($groupperms as $group => $gperm): if (!$groups->exists($groups->getGroupShortName($group))) continue; ?>
 <tr>
-  <td class="light"><?php echo htmlspecialchars($groups->getGroupName($group)) ?><input type="hidden" name="g_names[<?php echo htmlspecialchars($group) ?>]" value="<?php echo htmlspecialchars($group) ?>" /></td>
+  <td class="light"><?php echo htmlspecialchars($groups->getGroupShortName($group)) ?><input type="hidden" name="g_names[<?php echo htmlspecialchars($group) ?>]" value="<?php echo htmlspecialchars($group) ?>" /></td>
   <td align="center">
     <input type="checkbox" id="g_show_<?php echo htmlspecialchars($group) ?>" name="g_show[<?php echo htmlspecialchars($group) ?>]"<?php echo ($gperm & PERMS_SHOW) ? ' checked="checked"' : '' ?> />
     <label for="g_show_<?php echo htmlspecialchars($group) ?>" class="hidden"><?php echo _("Show") ?></label>
@@ -294,8 +294,8 @@
     <label for="g_names_new" class="hidden"><?php echo _("Select a group to add:") ?></label>
     <select id="g_names_new" name="g_names[||new]">
       <option value=""><?php echo _("Select a group to add") ?></option>
-    <?php foreach ($groupList as $gid => $group) { if (empty($groupperms[$gid])) { ?>
-      <option value="<?php echo htmlspecialchars($gid) ?>"><?php echo htmlspecialchars($group) ?></option>
+    <?php foreach ($groupList as $gid => $group) { if (empty($groupperms[$group])) { ?>
+      <option value="<?php echo htmlspecialchars($group) ?>"><?php echo htmlspecialchars($group) ?></option>
     <?php } } ?>
     </select>
   </td>
